distributed/dashboard/components/scheduler.py

```diff
@@ -4061,16 +4062,16 @@ def __init__(self, scheduler, start=None):
 
         self.root = Div(
             text=logs_html,
-            style={
-                "width": "100%",
-                "height": "100%",
-                "max-width": "1920px",
-                "max-height": "1080px",
-                "padding": "12px",
-                "border": "1px solid lightgray",
-                "box-shadow": "inset 1px 0 8px 0 lightgray",
-                "overflow": "auto",
-            },
+            # **{
+            #     "width": "100%",
+            #     "height": "100%",
+            #     "max-width": "1920px",
+            #     "max-height": "1080px",
+            #     "padding": "12px",
+            #     "border": "1px solid lightgray",
+            #     "box-shadow": "inset 1px 0 8px 0 lightgray",
+            #     "overflow": "auto",
+            # },
```

Comment:
I think you can now accomplish this similar to:

https://github.com/bokeh/bokeh/blob/branch-3.0/examples/basic/layouts/sizing_mode.py#L19

cc @mattpap for the best approach to support both 2.x and 3.x

Reply:
`UIElement.styles` is the replacement (works for all UI elements not just markup widgets). The name is `styles` and not `style`, so that people don't confuse this with DOM `style` attribute. I suppose we could have a deprecated alias in markup widgets for some time in 3.x.

Reply:
Thanks @mattpap! I think https://github.com/dask/distributed/pull/5648/commits/8d8f5f3107a97dbb24ba1851c6b8f770451c276a should address this one, though let me know if you think I've misunderstood your comment